content/user/ui/overview/Basic-quick-start-guide.md

```diff
@@ -0,0 +1,111 @@
+
+### How vulnerabilities work
+
+If a vulnerability is reported and Contrast has never seen it before, Contrast creates a new vulnerability. However, if that vulnerability already exists, Contrast updates the existing entry, issue count and last seen date.
+
+<a href="assets/images/Application-vulnerabilities-tab.png" rel="lightbox" title="Vulnerabilities in the application page"><img class="thumbnail" src="assets/images/Application-vulnerabilities-tab.png"/></a>
+
+> Example: This vulnerability was reported up to Contrast five times for one server. Instead of showing up as five vulnerabilities, Contrast updates this entry and increments the count. As Contrast continues to see the same findings, the count goes up. If you dive into the Notes tab within this vulnerability, you notice a list of the servers in which this vulnerability was found.
+ 
+## Improve Your Application Score
+
+Get your application secure by remediating vulnerabilities or enabling Protection rules. We provide you with a grade to show you how well your application is performing. Visit the [Contrast Scoring Guide](https://docs.contrastsecurity.com/user_tsguideapp.html#scoreguide) for more information. 
+ 
+## Track Libraries
+
+Be aware of libraries that may be vulnerable and bring them up to date by going to the Libraries tab in the Application page. 
+ 
+Contrast provides you with a grade for the library, known Common Vulnerabilities and Exposutres (CVEs), latest version and release date, used and total classes in the library, and the application that's using the library. Contrast calculates this grade based on three things: the age of the library, how many versions behind the library is, and the number of known CVEs that affect the library.
+
+For more information, read the article on [Library Analysis](https://docs.contrastsecurity.com/user_tsguidelib.html#analysis).
+ 
+## Manage Vulnerabilities
+
+Go the Vulnerabilities page to get details on each one and get rid of weaknesses so your application isn't compromised.
+ 
+### Analyze Findings
+
+Contrast discovers any code flaws, which are presented with a severity level to help you prioritize your tasks. For each reported vulnerability, you can mark them and create tags as needed. The following chart shows available statuses and behaviors when a vulnerability is marked and found again.
+
+---
+
+| Status        | Marked                          | Found Again          |
+|---------------|---------------------------------|----------------------|
+| Confirmed     | Stays Open                      | No Change            |
+| Suspicious    | Stays Open                      | No Change            |
+| Not a Problem | Closed - Requires Justification | Stays Closed         |
+| Remediated    | Closed                          | Reopened as **Reported** |
+| Reported      | Default                         | No Change            |
+| Fixed         | Closed                          | Stays Closed         |
+
+---
+
```

Comment:
"...dive into an application" isn't that descriptive. I would say "...drill into the application's Overview page..." or something a little more specific/consistent with how we reference that elsewhere.